var/spack/repos/builtin/packages/py-spdlog/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PySpdlog(PythonPackage):
+    """The py-spdlog package provides a Python wrapper
+    to the C++ spdlog library."""
+
+    homepage = 'https://github.com/bodgergely/spdlog-python'
+    url = 'https://github.com/bodgergely/spdlog-python/archive/v2.0.0.tar.gz'
+    git = 'https://github.com/bodgergely/spdlog-python.git'
+
+    # NOTE: We cannot use the archives of releases because they are missing
+    # the content of their spdlog directory, and they are not able to find
```

Comment:
If you use the PyPI tarball, it looks like it comes with the code we need. Then we can checksum the tarball properly.

Reply:
Alternatively, I could use the git tags instead of the specific commits.
There is only the 2.0.0 version on PyPi (and the links to archives are not super readable, with the full BLAKE2-256 hash in the URL).
What would be the preferred method?

Reply:
See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#url